__tests__/message-spec.js

```diff
@@ -11,11 +11,14 @@ describe('Message', () => {
   beforeEach(() => {
     testContext = {};
     testContext.connection = new NylasConnection('123', { clientId: 'foo' });
-    testContext.connection.request = jest.fn(() => Promise.resolve());
     testContext.message = new Message(testContext.connection);
     testContext.message.id = '4333';
     testContext.message.starred = true;
     testContext.message.unread = false;
+    testContext.message.to = [{"email": "foo", "name": "bar"}];
+    testContext.connection.request = jest.fn(() => {
+      return Promise.resolve(testContext.message.toJSON());
+    });
```

Comment:
nice change! little np: When the only statement in an arrow function is `return`, we can remove `return` and remove the surrounding curly brackets. So the syntax would look like:

```
testContext.connection.request = jest.fn(() => Promise.resolve(testContext.message.toJSON());
```